development/ts-migration-dashboard/scripts/write-list-of-files-to-convert.ts

```diff
@@ -30,30 +35,42 @@ async function main(): Promise<void> {
         );
       }),
     )
-  ).flat();
+  )
+    .flat()
+    .filter((filePath) => {
+      return !/^(?:\.storybook|node_modules)\//u.test(
+        path.relative(ROOT_DIRECTORY_PATH, filePath),
+      );
+    })
+    .sort();
   console.log(
     `Traversing dependency trees for ${entrypoints.length} entrypoints, please wait...`,
   );
-  const result = await madge(entrypoints, {
-    baseDir: ROOT_DIRECTORY_PATH,
-  });
-  const dependenciesByFilePath = result.obj();
-  const sortedFilePaths = Object.keys(dependenciesByFilePath)
-    .sort()
+
+  const dependenciesByFilePath = (
+    await madge(entrypoints, {
+      baseDir: ROOT_DIRECTORY_PATH,
+    })
+  ).obj();
+  const newFilePaths = Object.keys(dependenciesByFilePath)
     .filter((filePath) => {
       return (
-        /\.(?:js|tsx?)$/u.test(filePath) &&
-        !/^(?:\.storybook|node_modules)\//u.test(filePath)
+        /\.jsx?$/u.test(filePath) &&
+        !/^\./u.test(filePath) &&
+        // Filter this out again because some imports may refer to NPM modules
+        !/^node_modules\//u.test(filePath)
       );
-    });
+    })
+    .sort();
+  const updatedFilePaths = uniq(existingFilePaths.concat(newFilePaths));
```

Comment:
This line ensures that subsequent runs of `yarn ts-migration:enumerate` don't overwrite past runs.